intranet/apps/polls/models.py

```diff
@@ -94,6 +94,33 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return users.count()
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        percentage = 0
+        try:
+            percentage = voted / able
+        except ZeroDivisionError:  # there are no eligible voters
+            pass
+        return "{:.1%}".format(percentage)
```

Comment:
0 if able == 0 else voted / able
```suggestion
        return "{:.1%}".format(0 if able == 0 else voted / able)
```